.github/workflows/status-check.yml

```diff
@@ -11,7 +8,7 @@ jobs:
     strategy:
       matrix:
         # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
-        os: [ubuntu-20.04]
+        os: [ubuntu-20.04, windows-2019, macos-11]
```

Comment:
On this, I know Jack was working to get the tests passing on Windows, not sure if he completed that ticket though.
Not sure either on MacOS - yes, we should be testing there too, but I don't have a Mac, so it would be awkward for me to verify it works there (but I guess if Matthieu is making this PR, then it works for him there...?!).